java/src/by/gsu/epamlab/task1/Runner.java

```diff
@@ -0,0 +1,39 @@
+package by.gsu.epamlab.task1;
+
+import static by.gsu.epamlab.task1.BusinessTrip.convertToDecimal;
+
+public class Runner {
+    public static void main(String[] args) {
+        BusinessTrip[] trips = {
+                new BusinessTrip("test", 800, 3),
+                new BusinessTrip("test1", 1650, 7),
+                null,
+                new BusinessTrip("test3", 500, 2),
+                new BusinessTrip()
+        };
+
+        int maxTotal = 0;
```

Comment:
If I am not mistaken, our task required to display all the information about the trip.